shared/fs/banner/conflict-banner.tsx

```diff
@@ -5,35 +5,35 @@ import * as Types from '../../constants/types/fs'
 
 export type Props = {
   conflictState: Types.ConflictState
-  onStartResolving: () => void
-  onSeeOtherView: () => void
-  isUnmergedView: boolean
-  onFinishResolving: () => void
   onFeedback: () => void
+  onFinishResolving: () => void
+  onGoToTlf: (tlfPath: Types.Path) => void
   onHelp: () => void
+  onStartResolving: () => void
   tlfName: string
 }
 
-const getMessage = (tlf: string, conflictState: Types.ConflictState, isUnmerged): string => {
-  switch (conflictState) {
-    case Types.ConflictState.InConflictStuck:
+const getMessage = (tlf: string, conflictState: Types.ConflictState): string => {
+  switch (conflictState.type) {
+    case Types.ConflictStateType.Automatic:
+      return conflictState.isStuck
+        ? `Your changes to ${tlf} conflict with changes made to this` +
+            ' folder on another device. Automatic conflict resolution has failed,' +
+            ' so you need to manually resolve the conflict. This is not' +
+            ' supposed to happen!'
+        : 'This should not happen.'
+    case Types.ConflictStateType.ManualLocalView:
       return (
-        `Your changes to ${tlf} conflict with changes made to this` +
-        ' folder on another device. Automatic conflict resolution has failed,' +
-        ' so you need to manually resolve the conflict. This is not' +
-        ' supposed to happen!'
+        `You're resolving a conflict in ${tlf}. This is your local view.` +
+        'When you click finish, this view will go away. You should make ' +
+        'sure to copy any changes you want to keep into the global view.'
       )
-    case Types.ConflictState.InManualResolution:
-      return isUnmerged
-        ? `You're resolving a conflict in ${tlf}. This is your local view.` +
-            'When you click finish, this view will go away. You should make ' +
-            'sure to copy any changes you want to keep into the global view.'
-        : `This is the rest of the world's view of ${tlf}.` +
-            " When you're satisfied with this view, you can click finish."
-    case Types.ConflictState.Finishing:
-      return 'Finishing conflict resolution...'
-    case Types.ConflictState.None:
-    case Types.ConflictState.InConflictNotStuck:
+    case Types.ConflictStateType.ManualServerView:
+      return (
+        `This is the rest of the world's view of ${tlf}.` +
```

Comment:
This message changed in my PR because of the change to where you can click finish from - I can help rebase this after my PR is merged, if you like, since it's my changes causing the problems.

Reply:
I don't mind rebasing either; I guess depends on which one gets merged first.

Reply:
k. mine is merged now - lmk if rebase causes you issues